etc/nextcloudpi-config.d/nc-webui.sh

```diff
@@ -14,6 +14,7 @@
 #
 
 ACTIVE_=yes
+LANGUAGE_=[*auto*,en,de]
```

Comment:
can we find a different character other than `*`? as you well know we have to improve our shell backend, but until then I would rather only allow characters that are not special characters for bash

Reply:
Of, good catch. Any idea what to use? I couldn't name any character and be sure that it has no special meaning in bash (I'm not experienced enough with the language).
Possible candidates (that look fine in the last definition) are for example:
~ < > # % §

Reply:
let's use underscores `_` they are just like any other character

Reply:
Well okay. I don't like underscores so much (because they can't be used for the value itself anymore), but since it's only a temporary representation anyway, I'll just use them for now. :+1: